Servers/utils/assessment.utils.ts

```
@@ -18,28 +18,24 @@ export const getAssessmentByIdQuery = async (
   return result.rows.length ? result.rows[0] : null;
 };
 
-export const createNewAssessmentQuery = async (assessment: {
-  projectId: number;
-}): Promise<Object> => {
+export const createNewAssessmentQuery = async (assessment: Assessment): Promise<Object> => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)